kraken-build/src/kraken/std/python/tasks/publish_task.py

```diff
@@ -33,31 +33,28 @@ def get_relationships(self) -> Iterable[TaskRelationship]:
 
     def execute(self) -> TaskStatus:
         credentials = self.index_credentials.get()
```

Comment:
With
`interactive: Property[bool | None] = Property.default(None)`

```suggestion
    def execute(self) -> TaskStatus:
        # Check for the deprecated property
        if self.interactive.get() is not None:
            warnings.warn(
                "The 'interactive' property on the python.publish task is deprecated and has no effect. "
                "uv publish is non-interactive by default in this context.",
                DeprecationWarning,
            )
        credentials = self.index_credentials.get()
```